code/controllers/configuration/entries/game_options.dm

```diff
@@ -379,4 +379,8 @@
 	config_entry_value = 64
 	min_val = 0
 
+/datum/config_entry/number/ratcap
+	config_entry_value = 64
+	min_val = 0

```

Comment:
Since you add it here, it will work. You havent added it to game_options.txt though, if you want it to be adjustable you should do that or people will have to code dive to find the setting.